settings/src/components/account-status.js

```diff
@@ -65,9 +65,16 @@ export default function AccountStatus() {
 /**
  * Render a card for the status of the given setting.
  */
-function SettingStatusCard( { screen, status, headerText, bodyText } ) {
+function SettingStatusCard({ screen, status, headerText, bodyText }) {
+	const [ isFocused, setIsFocused ] = useState( false );
+	const className = [
+		"wporg-2fa__status-card",
+		"wporg-2fa__status-card-" + screen,
+		...( isFocused ? [ "wporg-2fa__status-card__focused" ] : [] ),
```

Comment:
This classname is used to apply `z-index` to the card (see [styles](https://github.com/WordPress/wporg-two-factor/pull/62/commits/1a46841a12fbae38f68af39d956561d8240d4ac5#diff-41c16ca64f385109d400c93a6d9fe13d0b8d2cab74a3381a361215a61ba72b94)), lifting it to the top of the stack and allowing the focus outline to be shown. The focus event fires on the nested anchor element so a state setter is passed down to be called on focus and blur.

Without the `z-index` it looks like this

![Screen Shot 2023-02-21 at 3 05 23 PM](https://user-images.githubusercontent.com/1017872/220229335-12077681-42ea-4368-a6fc-8e9769df670d.jpg)

Reply:
Would [`:focus-within`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) let us avoid the need to manually track the focus?

If not, another option might be moving the `<Card>` inside `ScreenLink.anchorText`.

Reply:
> Would [`:focus-within`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) let us avoid the need to manually track the focus?

Yep, works! TIL, thanks

Reply:
🎉 it looks much simpler now :)